content/axon-server-installation-guides/modules/ROOT/pages/installing_as_docker-k8s.adoc

```diff
@@ -0,0 +1,604 @@
+This image is more secure and useful in Kubernetes and OpenShift clusters.
+You should take care to declare the user- and group-id, both of which are `1001` and are named "`axonserver`".
+Doing this will ensure that any mounted volumes will be writable by the user running Axon Server.
+
+The images export the following volumes:
+
+* "`/axonserver/config`"
+
+This is where you can add configuration files, such as an additional `axonserver.properties` and the license file.
+However, you can also opt to use, for instance, Kubernetes, or Docker-compose secrets.
+Note that Axon Server EE assumes it can write to the directory configured with "`axoniq.axonserver.enterprise.licenseDirectory`", so you don't have to put the license on all nodes.
+* "`/axonserver/data`"
+
+This is where the ControlDB, the PID file, and a copy of the application logs are written to.
+* "`/axonserver/events`"
+
+In this volume the Event Store is created, with a single directory per context.
+* "`/axonserver/log`"
+
+In this volume the Replication Logs are created, with a single directory per Replication Group.
+* "`/axonserver/exts`"
+
+In this volume you can place Extension JAR-files, such as the LDAP and OAuth2 extensions.
+* "`/axonserver/plugins`"
+
+In this volume Axon Server will place all uploaded plugins.
+
+____
+
+_Using the axoniq/axonserver:latest image_ <a name="docker-latest"></a>
+
+Up to release 2023.2.0 the default Axon Server image (axoniq/axonserver:latest) was using Axon Server Standard Edition version 4.6.11. There is a difference in the directory structure between version 4.6.11 and the current version.
+One change is that the working directory has changed from `/` to `/axonserver`.
+All files and directories created by Axon Server are stored in this directory (or subdirectories).
+
+The following locations have new default values:
+
+|===
+|Property |Axon Server SE 4.6.11 |Axon Server 2023.2
+
+|axoniq.axonserver.event.storage |/eventdata |/axonserver/events
+|axoniq.axonserver.snapshot.storage |/eventdata |/axonserver/events
+|axoniq.axonserver.controldb-path |/data |/axonserver/data
+|axoniq.axonserver.plugin-package-directory |/data/plugins/bundles |/axonserver/plugins/bundles
+|axoniq.axonserver.plugin-cache-directory |/data/plugins/cache |/axonserver/plugins/cache
+|logging.file |/data/axonserver.log |/axonserver/data/axonserver.log
+|===
+
+As these locations have changed you need to update the volume mappings that you use in the container.
+The mapping for /eventdata must be changed to /axonserver/events, and the mapping for /data must be changed to /axonserver/data.
+
+____
+
+== Building you own image [[build-image]]
+
+A starter Dockerfile is included below which can be tailored as per your requirements.
+
+The starter file helps create the image in multiple stages,
+
+* The image will be based on a compact image from Google’s “distroless” base images at the gcr.io repository, in this case “gcr.io/distroless/java:11”.
```

Comment:
These list items could be moved right below the [source,bash] block and linked to actual lines in the code block with <.> 

See the "Bike Rental demo application structure" listing at https://raw.githubusercontent.com/AxonIQ/bike-rental-quick-start/main/docs/tutorial/modules/ROOT/pages/create-axon-framework-project.adoc